### PR TITLE
doc: working_with_nrf: squash structure for PMIC

### PR DESCRIPTION
**Pull request by greg-fer**
Reworked Working with PMIC pages.
Combined several pages into one. Renamed Working with to Developing with. NCSDK-22328.

### CODE REVIEW

File: samples/pmic/native/npm1300_fuel_gauge/README.rst [hunk 1]
```diff
@@ -16,16 +16,20 @@ The sample supports the following development kits:
 
 .. table-from-sample-yaml::
 
-The sample also requires an `nPM1300 EK <nPM1300 EK product page_>`_.
+The sample also requires an `nPM1300 EK <nPM1300 EK product page_>`_, which you need to connect to the development kit as described in `Wiring`_.
```
Comment (peknis) [inline samples/pmic/native/npm1300_fuel_gauge/README.rst:19]:
```suggestion
The sample also requires an `nPM1300 EK <nPM1300 EK product page_>`_ that you need to connect to the development kit as described in `Wiring`_.
```

File: samples/pmic/native/npm1300_one_button/README.rst [hunk 2]
```diff
@@ -17,20 +16,23 @@ The sample supports the following development kits:
 
 .. table-from-sample-yaml::
 
-The sample also requires an `nPM1300 EK <nPM1300 EK product page_>`_.
+The sample also requires an `nPM1300 EK <nPM1300 EK product page_>`_, which you need to connect to the development kit as described in `Wiring`_.
```
Comment (peknis) [inline samples/pmic/native/npm1300_one_button/README.rst:19]:
```suggestion
The sample also requires an `nPM1300 EK <nPM1300 EK product page_>`_ that you need to connect to the development kit as described in `Wiring`_.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/device_guides/pmic/npm1300_index.rst [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+.. _ug_npm1300_features:
+
+nPM1300 features
+****************
+
+nPM1300 is a PMIC with a linear-mode lithium-ion, lithium-polymer, and lithium ferro-phosphate battery charger.
+It has two dual-mode buck regulators, two dual purpose LDO/load switches, three LED drivers, and five GPIOs.
+
+nPM1300 is the perfect companion for nRF52 and nRF53 Series SoCs, and the nRF91 series SiP.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/device_guides/pmic/npm1300_index.rst:21]: @aasinclair , should we add nRF54L15 to the compatible boards list?

[the remark this answers is not in the document]
Comment (aasinclair) [reply doc/nrf/device_guides/pmic/npm1300_index.rst:21]: I think it's worth adding nRF54 series with this change.

**Review by shanthanordic — APPROVED** [review]

File: doc/nrf/device_guides/pmic/npm1300_index.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/device_guides/pmic/npm1300_index.rst]: @aasinclair , would this also mean that the nRF54 Series devices can be used with both nPM1300 samples in the NCS?

File: doc/nrf/device_guides/pmic/npm1300.rst [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+.. _ug_npm1300_features:
+
+nPM1300 features
+****************
+
+nPM1300 is a PMIC with a linear-mode lithium-ion, lithium-polymer, and lithium ferro-phosphate battery charger.
+It has two dual-mode buck regulators, two dual purpose LDO/load switches, three LED drivers, and five GPIOs.
+
+nPM1300 is the perfect companion for nRF52, nRF53, and nRF54 Series SoCs, and the nRF91 series SiP.
```
Comment (divipillai) [inline doc/nrf/device_guides/pmic/npm1300.rst:20]:
```suggestion
nPM1300 is the perfect companion for nRF52, nRF53, and nRF54 Series SoCs, and the nRF91 Series SiP.
```

File: doc/nrf/device_guides/pmic/npm1300.rst [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+It is ideal for compact and advanced IoT products such as wearables and portable medical applications.
```
Comment (divipillai) [inline doc/nrf/device_guides/pmic/npm1300.rst:21]:
```suggestion
It is ideal for compact and advanced IoT products, such as wearables and portable medical applications.
```

File: doc/nrf/device_guides/pmic/npm1300.rst [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+       | `User Guide <nRF52840 DK User Guide_>`_
+   * - :ref:`zephyr:nrf52dk_nrf52832`
+     - nPM1300 EK
+     - PCA10040
+     - ``nrf52dk/nrf52832``
+     - | `Product Specification <nRF52832 Product Specification_>`_
+       | :ref:`Getting started <ug_nrf52_gs>`
+       | `User Guide <nRF52 DK User Guide_>`_
+
+PMIC samples and libraries
```
Comment (divipillai) [inline doc/nrf/device_guides/pmic/npm1300.rst:83]: Should the samples and libraries section be moved towards the end of the documentation. I have see this is what we follow for at least for the integration template?

**Review by kylebonnici — APPROVED** [review]

**Review by aasinclair — APPROVED** [review]

File: doc/nrf/device_guides/pmic/npm1300.rst (hunk 6 above)
Comment (greg-fer) [reply doc/nrf/device_guides/pmic/npm1300.rst:84]: In this case, I think it makes sense to keep this section right after the compatible boards, given how building and programming for PMIC works -- you need a sample matching a board.

**Review by divipillai — APPROVED** [review]

**Pull request merged**